src/App.css

```diff
@@ -85,8 +83,9 @@ input {
 .cards {
   margin-bottom: 30px;
 }
-.card {
+.card1 {
```

Comment:
I know you changed it to avoid conflict with `card` in `itemCard.css`, but it'll be better if it had a meaningful name like `card-wrapper` for example.

Reply:
Everything looks great! Nice work, my friend @saswatsam786!